### ISSUE TITLE
Bhaskar.com screen reader's paused doesn't work 

### ISSUE THREAD

**Issue by chianger** [mozilla-tw/FirefoxLite#3225]
**Describe the bug**
Bhaskar.com screen reader's paused doesn't work 
(Testing with engineering build)

**Steps To Reproduce**
1. Go to 'https://www.bhaskar.com/breaking-news/news/india-surgical-strike-2-live-indian-airforce-mirage-enters-pakistan-occupied-kashmir-drops-bomb-6027422.html?ref=tpst-mfx' 

2. Click on the screen reader icon of the webpage
3. The icon becomes the pause icon, and audio readout begins
4. Click on the pause icon

**Expected behavior**
Screen reader pauses

**Actual behavior**
Audio continues

** Additional Info **
Chrome has the same behavior

**Device info (please complete the following information):**
 - Device: Pixel XL
 - OS: Android 8.1.0
 - Webview 72.0.3626.105
 - Version 1.1.0.debug.tyu

**Comment by chianger** [comment]
Reproduced with Chrome so it's probably the website's behavior.
